src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/apiUsage/debug/ForgottenDebugOutputInspector.java

```diff
@@ -110,6 +104,30 @@ private void recompileConfiguration() {
     }
 
     @NotNull
+    private static Set<String> optionConfigurationDefaults() {
+        final Set<String> migrated = new TreeSet<>();
```

Comment:
Why you are using `TreeSet` do we need to have the specific order of the functions FQN ? 
Maybe `HashSet` will be good here. 

https://stackoverflow.com/a/4464394/4053996 

I`m not good java developer;)